### ISSUE TITLE
Modal Accessibility 

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#64]
## Detailed description

> Describe in detail the issue you're having.
Looking at the modal dialog to help with accessibility issues. Looks it help us with our tabbing order and focus. However I noticed that when I do the zoom 200% test the content of the dialog are not visible. 

> Is this a feature request (new component, new icon), a bug, or a general issue?
Zoom accessibility issues.

> Is this issue related to a specific component?
modal

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

## Steps to reproduce the issue

1. Using the StoryBook, zoom out to 225% (use ctrl ++).

**Comment by alwilkin** [comment]
Can we get an ETA on fix?

**Comment by ninja511** [comment]
Moving to carbon-components repo and closing this one.
https://github.com/IBM/carbon-components/issues/1134
